diff --git a/lib/webpack/base.js b/lib/webpack/base.js
index 81f3336f5c5e36a5fe268c7bb20e273a8204cc00..c7fe3ff049e630056b1bb3aa5ffd6033b03da53c 100644
--- a/lib/webpack/base.js
+++ b/lib/webpack/base.js
@@ -52,6 +52,8 @@ async function createBaseConfig({ props, isServer, minify, }) {
     const mode = isProd ? 'production' : 'development';
     const themeAliases = await (0, aliases_1.loadThemeAliases)({ siteDir, plugins });
     return {
+        // [docs-site-patch] 降低 webpack 编译并发，控制大文档站编译内存峰值
+        parallelism: 2,
         mode,
         name,
         cache: {
